### PR TITLE
Doc tweaks

### PR DESCRIPTION
**Pull request by fernandezdavid**

### CODE REVIEW

File: pages/swapping/integrations/running-a-broker/affiliate-brokers.mdx [hunk 1]
```diff
@@ -10,14 +10,40 @@ import { Callout } from "@/components";
...
+
+This commission can be set at the time of deposit channel creation, which provides additional flexibility. Each affiliate receives their share out of all deposits made (and confirmed) while the deposit channel is open.
 
-See the [API documentation](broker-api#broker_request_swap_deposit_address) for further information.
 
 <Callout type="warning">
-Affiliate Brokers must be existing registered [Broker accounts](broker-api.mdx#2-registering-the-account).
-</Callout>
\ No newline at end of file
+Affiliate Brokers are still Broker accounts and so they must exist and be registered as [Broker accounts](broker-api.mdx#2-registering-the-account).
```
Comment (CumpsD) [inline pages/swapping/integrations/running-a-broker/affiliate-brokers.mdx:28]: drop the `.mdx` :)

File: pages/concepts/swaps-amm/just-in-time-amm-protocol.mdx [hunk 2]
```diff
@@ -23,7 +23,8 @@ Chainflip’s AMM design differs substantially from industry standards due to li
 
 - The JIT AMM is implemented directly on the State Chain in the '_pools_' pallet. Written in Rust, it's a substrate rewrite of the Uniswap v3 design on Ethereum with several changes and additions running in a custom execution environment.
 - **Range orders** work as they normally do in Uniswap v3. Liquidity remains deployed until the LP submits an extrinsic which removes it from the pool and returns the remaining funds and fees to their [liquidity account](liquidity-provider-accounts.md) balance.
-- **Limit orders** are layered on top of the existing range order system. Limit orders are fully consumed by swaps, and the resulting funds are returned directly to the LP's balance.
+- **Limit orders** are layered on top of the existing range order system. 
+- Both limit and range orders are consumed by swaps based on the best price at any given time. This means switching between range and limit orders, can occur an arbitrarily as needed. The resulting funds are returned to the LP's balance.
```
Comment (GabrielBuragev) [inline pages/concepts/swaps-amm/just-in-time-amm-protocol.mdx:27]:
```suggestion
- Both limit and range orders are consumed by swaps based on the best price at any given time. This means switching between range and limit orders, can occur arbitrarily as needed. The resulting funds are returned to the LP's balance.
```

File: pages/concepts/swaps-amm/just-in-time-amm-protocol.mdx [hunk 3]
```diff
@@ -52,17 +53,17 @@ As long as a few Market Makers compete with each other for the trades, this prot
...
-- Chainflip requires 6 Ethereum blocks to consider a deposit transaction ‘final.’ It is therefore estimated that the swap will be executed in 14 Chainflip State Chain blocks (around 90 seconds). Market Makers now must include any State Chain limit order or range order updates in those 14 blocks to compete for the trade.&#x20;
-- By having additional capital float on other exchanges like Coinbase, Binance, and so on, Market Makers can use risk model calculations to determine their best possible price for the trade. Using this calculation, they update their range orders and move their BTC in the BTC-USDC pool right up to that price. In this case, Market Maker A creates a sell limit order worth \$10,000 USDC at \$23,934 per BTC, whereas Market Maker B moves their BTC sell order to \$23,938 per BTC.&#x20;
+- Chainflip requires 7 Ethereum blocks to consider a deposit transaction ‘final.’ It is therefore estimated that the swap will be executed in 14 Chainflip State Chain blocks (around 90 seconds). Market Makers now must include any State Chain limit order or range order updates in those 14 blocks to compete for the trade.
```
Comment (GabrielBuragev) [inline pages/concepts/swaps-amm/just-in-time-amm-protocol.mdx:61]:
```suggestion
- Chainflip requires 7 Ethereum block confirmations to consider a deposit transaction ‘final.’ It is therefore estimated that the swap will be executed in 14 Chainflip State Chain blocks (around 90 seconds). Market Makers now must include any State Chain limit order or range order updates in those 14 blocks to compete for the trade.
```

File: pages/concepts/validator-network/ingress-witnessing-deposits.mdx [hunk 4]
```diff
@@ -29,10 +29,19 @@ As an essential function of the settlement system, Validators are required to ma
...
-Once a related transaction to an address or contract on the “watchlist” occurs on the external chain, the Validators wait until a pre-specified confirmation window has passed before Witnessing the event. The exact length of the confirmation window (measured in blocks) is assessed on a case-by-case basis for supported blockchains.
+Once a related transaction to an address or contract on the “watchlist” occurs on the external chain, the Validators wait until a pre-specified confirmation window has passed before Witnessing the event. The length of the confirmation window (measured in blocks) is known as **safety margin**.
 
 > No decentralised interoperability protocol can escape the reality of blockchain finality and are uniquely vulnerable to rollbacks on other chains, thus resulting in an inevitable delay before witnessing and processing can occur.
 
+The safety margin varies by chain. Some chains, like Polkadot, have deterministic finality and do not require any confirmations other than being included in a block.
+
+|Blockchain| Safety Margin      | Block Duration |
+|----------|--------------------|----------------|
+|Bitcoin   |                   4| ~10 min        |
```
Comment (GabrielBuragev) [inline pages/concepts/validator-network/ingress-witnessing-deposits.mdx:40]:
```suggestion
|Bitcoin   |                   3| ~10 min        |
```

File: pages/swapping/intro-to-boost.mdx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+### Example: Boost BTC to ETH swap
+
+1. A user states the intention to swap a given amount of BTC to ETH with *Boost* feature enabled
+2. The protocol [opens a *boosteable* Swap Deposit Channel](../concepts/swaps-amm/deposit-channels-and-brokers#opening-deposit-channels) and generates a Bitcoin deposit address
+3. The user sends the desired amount of BTC to the generated deposit address
+3. The protocol attempts to [*Boost* the incoming deposit](../lp/boost/how-boost-works#boosting-a-deposit), by checking the available *Boost* liquidity provided by LPs
+4. If there is enough collateral in the *Boost* pools, the protocol pre-witness the deposit **after it has been included in a block**, minus a small *Boost* fee
+5. As soon as the [deposit is witnessed](../concepts/validator-network/ingress-witnessing-deposits#witnessing), the swap is triggered by the [JIT AMM](../concepts/swaps-amm/just-in-time-amm-protocol), and [sends the resulting ETH](../concepts/validator-network/egress-broadcasting-funds) to the destination address
+6. The user receives ETH, saving ±20 minutes of total swap time
```
Comment (GabrielBuragev) [inline pages/swapping/intro-to-boost.mdx:64]:
```suggestion
4. The protocol attempts to [*Boost* the incoming deposit](../lp/boost/how-boost-works#boosting-a-deposit), by checking the available *Boost* liquidity provided by LPs
5. If there is enough collateral in the *Boost* pools, the protocol boosts the deposit **after it has been included in a block (pre-witnessed)**, deducting a small *Boost* fee from the total deposited amount
6. As soon as the deposit is pre-witnessed and boosted, the swap is triggered by the [JIT AMM](../concepts/swaps-amm/just-in-time-amm-protocol), and [sends the resulting ETH](../concepts/validator-network/egress-broadcasting-funds) to the destination address
7. The user receives ETH, saving ±20 minutes of total swap time
```

File: pages/swapping/intro-to-boost.mdx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+<Callout type="warning">
+If there isn't enough collateral in the *Boost* pools, the protocol defaults to witness the deposit with regular block confirmations and **no *Boost* fee is applied**.
+</Callout>
+
+## Considerations
+
+- The protocol will attempt to Boost *any* amount sent after one block confirmation, if Boost liquidity is available.
+- The user's intended amount acts as an upper limit; anything less increases the chances of a successful boost
```
Comment (GabrielBuragev) [inline pages/swapping/intro-to-boost.mdx:73]: I don't think we need this as it is not accurate.
```suggestion

```

File: pages/concepts/validator-network/ingress-witnessing-deposits.mdx (hunk 4 above)
Comment (fernandezdavid) [reply pages/concepts/validator-network/ingress-witnessing-deposits.mdx:40]: then I guess https://docs.chainflip.io/swapping/integrations/javascript-sdk/get-required-block-confirmations#sample-response is outdated too @GabrielBuragev ?

Comment (GabrielBuragev) [reply pages/concepts/validator-network/ingress-witnessing-deposits.mdx:40]: Yeah it is. I will update that in my PR

**Pull request merged**